src/node_mgmt/env_scan.py

```diff
@@ -368,6 +374,7 @@ def do_scan(self):
         except Exception:
             logger.exception("Exception while running ModbusTCP scan")
         serial_devices = self.serial_env.serial_scan()
+        speedwire_serials = self.speedwire_scan()
```

Comment:
[2/2] ...instead, we can just do `speedwire_serials = self.speedwire_env.scan_serials()` here. Like we do with the serial scan.